lib/matplotlib/axes/_base.py

```diff
@@ -3096,9 +3096,9 @@ def _adjust_ylim(self):
                                                               ys=ys,
                                                               xlim=xlim
                                                              )
-                if not new_miny is None and (miny is None or new_miny < miny):
+                if new_miny is not None and (miny is None or new_miny < miny):
```

Comment:
The diff is pep8 check complaint.
But if you ask if "if new_miny is not None" can be replaced by "if new_miny", it can not, because if new_miny == 0, that would result in False. And that is not what we want there.